templates/istio.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: istio
+  labels:
+    kubeaddons.mesosphere.io/name: istio
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.3.3-1"
+    appversion.kubeaddons.mesosphere.io/istio: "1.3.3"
+    endpoint.kubeaddons.mesosphere.io/kiali: "/ops/portal/kiali"
```

Comment:
I'm curious, is this a new annotation type?

Reply:
I copied it from other YAMLs. I am under impression that it is used by the UI team but I could very well be wrong here :).